src/stavenote.ts

```diff
@@ -133,9 +133,10 @@ export class StaveNote extends StemmableNote {
     const notesList: StaveNoteFormatSettings[] = [];
 
     for (let i = 0; i < notes.length; i++) {
-      const props = notes[i].getKeyProps();
-      const line = props[0].line;
-      let minL = props[props.length - 1].line;
+      // formating relys on sortedKeyNotes in order to calculate line & minL
```

Comment:
Typo:

``
`
// Formatting uses sortedKeyProps to calculate line and minL.
```

Reply:
Done directly on master.